### ISSUE TITLE
About the metric MSE

### ISSUE THREAD

**Issue by llan-ml** [yunshengb/GraphSim#3]
Hi @yunshengb , thanks for releasing this repo.

I found that you use a denominator of 2 in the function of [`mean_squared_error`](https://github.com/yunshengb/SimGNN/blob/2abc5a367e7ac0a5779d1979cbb510c53a4a15f9/src/metrics.py#L69-L78), which I don't think should be used when reporting the metric of MSE.

It would be better to declare this in the readme file or update the code.

**Comment by yunshengb** [comment]
Thanks for your note. Indeed, we use a denominator 2 in the evaluation. We encourage you to reuse our code for reproducing the results, and please let us know any other confusion or questions you have. Thanks!

**Comment by llan-ml** [comment]
@yunshengb 
Thanks for your reply.

I have another question about the training convergence. I found that the convergence results across different runs with the same configuration vary a lot as shown in the following figure:
![image](https://user-images.githubusercontent.com/22030149/102099966-db7cb680-3e63-11eb-936f-4d99f9d9ba07.png)

I guess that the reason could be different runs are with different random seeds, which further influences the sampling of training data and the initialization of weights. However, the variance across different runs is quite big.

Do you also encounter this problem and do you have any advice to alleviate this?

**Comment by rishabh-ranjan** [comment]
> Thanks for your note. Indeed, we use a denominator 2 in the evaluation. We encourage you to reuse our code for reproducing the results, and please let us know any other confusion or questions you have. Thanks!
> […](#)

**Why** is the denominator of 2 used in the evaluation? This has not been documented in the paper. My concern is that the numbers in the paper under the heading of MSE are misleading, since the standard definition of MSE does not have this factor of 1/2. Please clarify.
